build.gradle

```diff
@@ -2373,7 +2373,7 @@ project(':connect:runtime') {
     testImplementation libs.junitVintageEngine
     testImplementation libs.powermockJunit4
     testImplementation libs.powermockEasymock
-    testImplementation libs.mockitoCore
+    testImplementation libs.mockitoInline // supports mocking static methods, final classes, etc.
```

Comment:
Why are we switching to `mockitoInline`? We don't seem to use any of these feature in this PR and according to the Mockito project, inline may be removed in the future when all of its features are integrated in core.